BoxContentSDK/BoxContentSDK/Requests/BOXRequest_Private.h

```diff
@@ -27,6 +27,12 @@
                subresource:(NSString *)subresource
                      subID:(NSString *)subID;
 
+- (NSURL *) URLWithResource:(NSString *)resource
```

Comment:
Minor: I'd move this above the other just to keep the expansion of parameters consistent for when we're reading through it.